packages/core-ethereum/src/index.mock.ts

```diff
@@ -1,51 +1,52 @@
-import { AccountEntry, Address, debug, NativeBalance } from '@hoprnet/hopr-utils'
+import { AccountEntry, debug, NativeBalance, PublicKey } from '@hoprnet/hopr-utils'
 import type HoprCoreEthereum from '.'
 import BN from 'bn.js'
 import PeerId from 'peer-id'
 import { Multiaddr } from 'multiaddr'
 
-export const sampleAddress = Address.fromString('0x55CfF15a5159239002D57C591eF4ACA7f2ACAfE6')
-export const samplePeerId = PeerId.createFromB58String('16Uiu2HAmThyWP5YWutPmYk9yUZ48ryWyZ7Cf6pMTQduvHUS9sGE7')
-export const sampleMultiaddrs = new Multiaddr(`/ip4/127.0.0.1/tcp/124/p2p/${samplePeerId.toB58String()}`)
-
 const connectorLogger = debug(`hopr:mocks:connector`)
-const connectorMock = {
-  start: () => {
-    connectorLogger('starting connector called.')
-    return {} as unknown as HoprCoreEthereum
-  },
-  stop: () => {
-    connectorLogger('stopping connector called.')
-    return Promise.resolve()
-  },
-  getNativeBalance: () => {
-    connectorLogger('getNativeBalance method was called')
-    return Promise.resolve(new NativeBalance(new BN('10000000000000000000')))
-  },
-  getPublicKey: () => {
-    connectorLogger('getPublicKey method was called')
-    return {
-      toAddress: () => Promise.resolve(sampleAddress)
+function createConnectorMock(peer: PeerId): HoprCoreEthereum {
```

Comment:
Properly mock connector such that e.g. `.getPublicKey()` and `.getAccount()` works. Necessary to debug whether e2e test nodes are indeed funded.